shell/platform/darwin/ios/ios_render_target_gl.mm

```diff
@@ -87,20 +98,23 @@
   glDiscardFramebufferEXT(GL_FRAMEBUFFER, sizeof(discards) / sizeof(GLenum), discards);
 
   glBindRenderbuffer(GL_RENDERBUFFER, colorbuffer_);
-  return [[EAGLContext currentContext] presentRenderbuffer:GL_RENDERBUFFER];
+  auto current_context = [EAGLContext currentContext];
```

Comment:
nit: this might be a source of slowness, querying the current context each time we present.  No need to change anything, just flagging it.